openslides/motions/serializers.py

```diff
@@ -314,6 +315,12 @@ class Meta:
             'read_groups',
             'write_groups',)
 
+    def create(self, validated_data):
```

Comment:
Can you please explain this change? Creating a section should trigger inform_changed_data so why do we have to call it manually again?

Reply:
This was strange: On creation (even with groups) I got as autoupdate always empty arrays for read and write groups. The tests we have worked fine and I looked in the DB and the groups were set there. With this, the cache gets updated, and that was not happen before. I suspect, that setting things on m2m relations (like user.groups.add(2)) does not trigger an autoupdate and not updating the cache.

If you have a workaround or a nicer place for this `inform_changed_data` please give a hint..